packages/fiori/src/NotificationListGroupItem.hbs

```diff
@@ -0,0 +1,80 @@
+<li
+	class="ui5-nli-group-root ui5-nli-focusable"
+	@focusin="{{_onfocusin}}"
+	@focusout="{{_onfocusout}}"
+	@keydown="{{_onkeydown}}"
+	role="option"
+	tabindex="{{_tabIndex}}"
+	dir="{{rtl}}"
+	aria-labelledby="{{ariaLabelledBy}}"
+>
+	<div class="ui5-nli-group-header">
+		<ui5-button
+			icon="navigation-right-arrow"
+			design="Transparent"
+			@click="{{_onBtnToggleClick}}"
+			class="ui5-nli-group-toggle-btn"
```

Comment:
I think that you only need either the class or the toggle-btn attribute. As much as I can see you are using both of them to identify this exact DOM node. Moreover you are using the same attribute in ```NotificationListItetm.hbs```. Probably you could go with the class only?